src/ida_lunde_naalsund_ex/ex_02/bubble_sort.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+
+def bubble_sort(data):
+    """Takes a list or tuple of numbers as input.
+    Returns a copy of the list where the numbers are
+    sorted in increasing order.
+
+    :param data: List or tuple containing numbers.
+    :return: Sorted list.
+    """
+    copy = list(data)
+    n = len(copy)
+
+    while n > 0:
```

Comment:
cool approach, but consider using for-loops instead and look at how the `range()` object can be used to decrement and increment from- and to various integers.